lib/digit_calculator.ex

```diff
@@ -11,6 +11,13 @@ defmodule BRAN.DigitCalculator do
     |> rem(mod_factor)
   end
 
+  @spec mod_simple([Integer.t()], Integer.t(), [Integer.t()], Function.t()) :: Integer.t()
```

Comment:
Não existe Integer.t() e nem Function.t(), segue correção:

```suggestion
  @spec mod_simple([integer()], integer(), [integer()], function()) :: integer()
```
Referência: https://hexdocs.pm/elixir/typespecs.html#basic-types